GameData/RealSolarSystem/RSSKopernicus/Uranus/Uranus.cfg

```diff
@@ -125,6 +125,22 @@
 				}
 			}
 		}
+		
+		Rings
+		{
+			Ring
+			{
+				angle = 74.8989043025419
```

Comment:
At least in my principia install, this angle puts the rings out of alignment with all the moons, as well as with the planet's rotation angle. angle=0 gives the expected result.

Reply:
If that's the case then you need to add another patch with `NEEDS[Principia]` and the different angle value